src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -29,32 +30,36 @@ export type ActionName =
     | "Download Driver Overview"
     | "Delete Parcel";
 
-type AvailableActionsType = {
-    [actionKey in ActionName]: {
-        newStatus: StatusType;
-    };
+type ActionTypes = {
+    actionName: ActionName;
+    newStatus: StatusType;
+    availableToVolunteers: boolean;
 };
 
-export const availableActions: AvailableActionsType = {
-    "Download Day Overview": {
+const availableActions: ActionTypes[] = [
```

Comment:
Nice well done, I like this a lot! This makes it much easier to add a new action, see which actions are restricted to volunteers, and we're not duplicating newStatus. Also we don't need to do Object.entries, and [key] unnecessarily which makes it more readable :)